docs/modules/ROOT/pages/client-auth.adoc

```diff
@@ -0,0 +1,109 @@
+= Client Authentication and Authorization
+
+Use the Hazelcast Platform Operator to configure xref:hazelcast:security:authentication-overview.adoc[Client Authentication] and xref:hazelcast:security:client-authentication.adoc[Client Authentication].
+
+== Simple Authentication
+
+You can easily use xref:hazelcast:security:simple-authentication.adoc[Client Authentication][Hazelcast Simple Authentication] by making minimal changes in the original configuration.
```

Comment:
Should this be '[Client Authentication] and [Hazelcast Simple Authentication]' rather than '[Client Authentication][Hazelcast Simple Authentication]'? 
(I'm not too familiar with the product so apologies if this is a silly question)

Reply:
Fixed this one as well, would you check again?